src/modules/ROOT/pages/changelog/202303/01.json

```diff
@@ -0,0 +1,889 @@
+    }
+  ],
+
+  "fixed": [
+    {
+      "what": [
+        "the sidebar in the reporter webapp not opening on hover."
+      ],
+      "how": [
+        "by removing the disabledRoutes() watcher which was causing this issue."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/e3b1ec8022324003fa22b7be6803fdf89021c929"
+      ]
+    },
+    {
+      "what": [
+        "saving issue with module discovery settings.",
+        "If you had any configuration set for discovery in a module, the module was not saving properly."
+      ],
+      "how": [
+        "by Migrating discovery settings to support multi-language, and updated module discovery config struct to support multi lang result."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/ab283f89c38534838b183834b4b13893dadd302e",
+        "https://github.com/cortezaproject/corteza/commit/3af167c1d60237986cf11da9d197ac526a349683"
+      ]
+    },
+    {
+      "what": [
+        "discovery search result with namespace, module filters."
+      ],
+      "how": [
+        "by adding minimum string match criteria to match exact string match for search results."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/e2c374c106f13455811fce83bde2ed654e8e964a"
+      ]
+    },
+    {
+      "what": [
+        "post processing function in Gauge chart in Compose not updating accordingly."
+      ],
+      "how": [
+        "by adding a post processing method to Gauge Chart compose type."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/865/commits/07acfa78df43d56fc4db27659561a0ad06cc49b7"
+      ]
+    },
+    {
+      "what": [
+        "long page titles being cut off."
+      ],
+      "how": [
+        "by truncating long titles to prevent cut off."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1069/commits/583a4e8b994497049adc17fb064dfca4806cd6f8"
+      ]
+    },
+    {
+      "what": [
+        "record add or update error when dedicated record revisions table is configured."
+      ],
+      "how": [
+        "by rearranging the order of the rel_resource, revision, and operation columns when custom dedicated record revision table is configured and changed the operation column datatype to text."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/6b958e0edbbe450407e4a779c3dc3eaff3a8d916"
+      ]
+    }
+  ],
+
+  "removed": [
+    {
+      "what": [
+        "the bootstrap class text-truncate from the <b-list-group-item>."
+      ],
+      "why": [
+        "because the federation nodes in modules were not shown as they should and this class was making the problem."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/8b7f903378a867d193fb4a20a8a1fefe8d2db60e"
+      ]
+    },
+    {
+      "what": [
+        "relative value option from bar and line charts."
```

Comment:
I'd put this under changed.
Fixed is more for removed features/elements of the thing, like removing an API endpoint or a specific page block.
IMO, hiding some config options that _never even worked_ wouldn't fit in here.